### DISCUSSION TITLE
Page for monitoring

### DISCUSSION THREAD

**Discussion by Giuseppe-TD** [isontheline/pro.webssh.net#950] [Ideas]
I'like if in The future will be added a Page/section for monitoring a server like servercat has.
A Page with statistics mem used cpu used with graphics:)

**Comment by isontheline** [comment]
Hello @Raxiel1987 👋

Thank you so much for your feedback 🙏

I'm currently working on it : 
* #910
* https://github.com/isontheline/pro.webssh.net/issues/941

**Reply by Giuseppe-TD** [reply]
Ohh good thank you so much

Happy to have bought The full version 😁

**Reply by isontheline** [reply]
Thank you a lot for your support 🙏😉
